setup.cfg

```diff
@@ -4,7 +4,7 @@ ignore =
     bootstrap.py
 
 [flake8]
-ignore = C901,N801,N802,N803,N805,N806,N812,E301
+ignore = C901,N801,N802,N803,N805,N806,N812,E301,P001
```

Comment:
`flake8-plone-api` should not be used on this repos, so ignoring `P001` should not be necessary.